frontend/src/pages/vault/stake/Stake.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import STRK from '../../../assets/icons/strk.svg';
+import USDC from '../../../assets/icons/borrow_usdc.svg';
+import './stake.css';
+import { VaultLayout } from '../VaultLayout';
+import { CogIcon } from 'lucide-react';
+
+function Stake() {
+  const [selectedNetwork, setSelectedNetwork] = useState('Starknet');
+
+  const networks = [{ name: 'Starknet', image: STRK }];
+
+  const handleChange = (e) => {
+    setSelectedNetwork(e.target.value);
+  };
+  return (
+    <VaultLayout>
+      <div className="staking-container">
+        <div className="balance-row">
+          <div className="balance-card">
```

Comment:
please use different name, not `balance-card` it conflicts with some items on other pages, and we'll deal with it later. for now just name it differently

Reply:
still unresolved